docs/graph-homomorphism-as-natural-transformation.md

```diff
@@ -5,10 +5,14 @@
  * Given a functor G from C to D.
  
 There is a natural transformation from F to G if, for every object X in category C, there is an endomorphism from F(X) to G(X).
+[Paul: Two points:
+1. The word "endomorphism" usually means a morphism from an object to itself.  In this case F(X) and G(X) are generally distinct, so you would just have a "morphism" from F(X) to G(X) rather than an "endomorphism".
```

Comment:
OK so my understanding of endomorphism was that it was a morphism within a category, i.e. given category X with objects a and b a morphism from a to b is an endomorphism. A functor is a mapping between categories and I thought that a morphism was a functor - the main difference being that a morphism being something that applies to many branches of mathematics and a functor being defined in category theory only.

I think I need to be more precise and consistent with my terminology.

How does an endomorphism as you describe it differ from the identity morphism?